examples/system/cpp_pthread/main/cpp_pthread.cpp

```diff
@@ -0,0 +1,109 @@
+#include <iostream>
+#include <thread>
+#include <chrono>
+#include <memory>
+#include <string>
+#include <sstream>
+#include <esp_pthread.h>
+#include <freertos/FreeRTOS.h>
+#include <freertos/task.h>
+#include <esp_log.h>
+
+using namespace std::chrono;
+
+const auto sleep_time = seconds{5};
+
+void print_thread_info(const char* extra = nullptr)
+{
+        std::stringstream ss;
+        if(extra)
```

Comment:
Tabs, space between if and paren, opening brace should be on the same line with the condition.

(Or may just run the file through the formatting script)

Reply:
I looked at the contributing.rst file at github, but links in those files are broken when viewed there. Didn't realize that the docs.espressif.com shows them correctly. I'll format the file with the script.